quickjs.c

```diff
@@ -35519,11 +35420,7 @@ static int JS_WriteBigInt(BCWriterState *s, JSValueConst obj)
     bc_put_leb128(s, len);
     if (len > 0) {
         for(i = 0; i < (len / (JS_LIMB_BITS / 8)); i++) {
-#if JS_LIMB_BITS == 32
```

Comment:
I guess this meant the BC wasn't compatible across architectures?

Reply:
Yes, indeed.